### ISSUE TITLE
Include a license file

### ISSUE THREAD

**Issue by RohanKPOfficial** [Nothing-Developer-Programme/Glyph-Developer-Kit#5]
Please include a license file and acknowledge if developers are free to include the assets in this sdk as part of their applications.

**Comment by CrazyMarvin** [comment]
Yes, a **LICENSE** is needed so that [Flashy ](https://github.com/Crazy-Marvin/Flashy/issues/205)may implement this.
